client/src/Components/NavBar/Navbar.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import "./Navbar.css";
+import * as FaIcons from "react-icons/fa";
+import * as AiIcons from "react-icons/ai";
+import { Link } from "react-router-dom";
+import { NavbarData } from "./NavbarData";
+
+function Navbar() {
+   {/* Setting state on sidebar to aid in open/close*/}
+  const [sidebar, setSidebar] = useState(false);
+
+  const showSidebar = () => setSidebar(!sidebar);
+  return (
+    <div id="navbar">
+      <Link to="#" id="menu-bars">
+        <FaIcons.FaBars onClick={showSidebar} />
+      </Link>
+       {/*Menu Toggle active/off . Menu Items will display or slide left*/}
+      <nav id={sidebar ? "nav-menu active" : "nav-menu"}>
```

Comment:
You should add an something so that when the user either leaves the area of the drop down (with their mouse) or clicks on one of the links, the sidebar box should close. It would make the UI better,

Also using a semi transparent background makes the text harder to read with your sidebar overlapping the text on the page